src/WatcherCommand.php

```diff
@@ -38,9 +41,26 @@ protected function determineOptions(InputInterface $input): array
         return $options;
     }
 
+    protected function getOptionsConfigFile()
+    {
+        $configPath = getcwd();
+        $configFile = $configPath . self::PHPUNIT_WATCH_CONFIG_FILENAME;
+
+        while (! file_exists($configFile)):
+            $configPath = dirname($configPath);
+            $configFile = $configPath . self::PHPUNIT_WATCH_CONFIG_FILENAME;
+            if ($configPath === '/') {
+                $configFile = self::PHPUNIT_WATCH_CONFIG_FILENAME;
+                break;
+            }
```

Comment:
Inside the while, if a config file is found, immediately read it and return the config values.